stdlib/public/core/VarArgs.swift

```diff
@@ -60,7 +60,7 @@ protocol _CVarArgAligned : CVarArg {
 
 #if arch(x86_64)
 @_versioned
-internal let _x86_64CountGPRegisters = 6
+internal let _countGPRegisters = 6
```

Comment:
I think that the name here could be better as `Max?Registers` or `Saved?Registers`.

Reply:
`_countGPRegisters` is the general register number of the platform;